src/test/resources/nonPen.tsv

```diff
@@ -0,0 +1,3 @@
+Gene
```

Comment:
not all columns have a column header. can we add column headers for all columns to make it a 'more standard' .tsv?